### PR TITLE
[3.2] Add documentation on Code Snippets

### PR DESCRIPTION
**Pull request by svdimitr**

### CODE REVIEW

File: getting-started/vs-code-integration/snippets.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
+---
+title: Telerik UI for Blazor Code Snippets
+page_title: Telerik UI for Blazor Code Snippets - Visual Studio Code Integration
+description: Learn how to add a new component  Templates.
```
Comment (dimodi) [inline getting-started/vs-code-integration/snippets.md:4]:
```suggestion
description: Learn how to add a new component in Visual Studio Code via code snippet templates.
```

File: getting-started/vs-code-integration/snippets.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+slug: getting-started-vs-code-integration-new-project
```
Comment (dimodi) [inline getting-started/vs-code-integration/snippets.md:5]: Are we sure about this slug name? It doesn't appear to reflect the article content.

File: getting-started/vs-code-integration/snippets.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+The Telerik Visual Studio Code extension provides a handy feature for increased developer productivity – code snippets for fast UI component reference and configuration. 
+
+## Get the Extension
+
+To use the **Telerik UI for Blazor Code Snippets**, install the Telerik UI for Blazor Visual Studio Code Extension. @[template](/_contentTemplates/common/general-info.md#vs-code-x-download)
+
+## Add a Code Snippet to a .razor page
+
+1. Use the `tb` shortcut that stands for Telerik Blazor and select a snippet from the dropdown menu.
+    1. Alternatively, you can type the name of the component and the available snippets will appear in a dropdown menu.
```
Comment (dimodi) [inline getting-started/vs-code-integration/snippets.md:21]:
```suggestion
    1. Alternatively, type the name of the component and the available snippets will appear in a dropdown menu. 
```

File: getting-started/vs-code-integration/snippets.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+| tb-treelist                | [TreeList]({%slug treelist-overview%})   |
+| tb-treeList-editable       | [TreeList with Editing]({%slug treelist-editing-overview%}) |
+| tb-treelistcolumn          | [TreeList - Databound Column]({%slug treelist-columns-bound%}) |
+| tb-treelistcolumn-locked   | [TreeList with Frozen Column]({%slug treelist-columns-frozen%}) |
+| tb-treeview                | [TreeView]({%slug treeview-overview%})   |
+| tb-upload                  | [Upload]({%slug upload-overview%})       |
+| tb-window                  | [Window]({%slug components/window/overview%}) |
+| tb-wizard                  | [Wizard]({%slug wizard-overview%})       |
+
+
```
Comment (dimodi) [inline getting-started/vs-code-integration/snippets.md:108]:
```suggestion
```

File: getting-started/vs-code-integration/snippets.md (hunk 2 above)
Comment (svdimitr) [reply getting-started/vs-code-integration/snippets.md]: Agreed, I accidentally deleted the first article and had to create a new one and forgot the slug, fixed

File: getting-started/vs-code-integration/snippets.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
+---
+title: Telerik UI for Blazor Code Snippets
+page_title: Telerik UI for Blazor Code Snippets - Visual Studio Code Integration
+description: Learn how to add a new component in Visual Studio Code via code snippet templates.
+slug: code-snippets-vs-code
```
Comment (dimodi) [inline getting-started/vs-code-integration/snippets.md:5]: Let's follow the current convention -

```suggestion
slug: getting-started-vs-code-integration-snippets
```

**Review by dimodi — APPROVED** [review]

**Pull request merged**